templates/press-centre/index.html

```diff
@@ -0,0 +1,106 @@
+{% extends "base_index.html" %}
+
+{% block title %}Press centre{% endblock %}
+
+{% block meta_description %}The home of Canonical media resources, news stories and press releases.{% endblock %}
+
+{% block content %}
+  <section class="p-strip--standard-dark is-dark is-shallow">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7 p-card--suru">
+        <h1 class="p-heading--1">Press centre</h1>
+        <p class="p-heading--4">The home of Canonical media resources, news stories and press releases. Journalists seeking further information can contact <a class="p-link--inverted" href="mailto:pr@canonical.com">pr@canonical.com</a>
+        </p>
+      </div>
+      <div id="rtp-banner" class="rtp-banner"></div>
+    </div>
+  </section>
+
+  <section class="p-strip is-deep" id="posts-list">
+    <div class="row">
+      <div class="col-8">
+        {% for article in articles %}
+          {% include 'press-centre/card.html' %}
+        {% endfor %}
+      </div>
+
+      <div class="col-4">
+        <div class="p-card">
+          <h4>Logos</h4>
+          <ul class="p-list">
+            <li class="p-list__item is-ticked" style="background-image:url('data:image/svg+xml,%3Csvg xmlns=%22http://www.w3.org/2000/svg%22 width=%2214%22 height=%2214%22 viewBox=%220 0 14 14%22%3E%3Ccircle fill=%22%23333%22 cx=%227%22 cy=%227%22 r=%227%22/%3E%3Cpath fill=%22%23fff%22 d=%22M6.1 10.813L2.41 8.105l1.184-1.613L5.9 8.187l4.393-4.394 1.414 1.414z%22 /%3E%3C/svg%3E');"><a href="https://assets.ubuntu.com/v1/18a64171-ubuntu-brandmark1.zip" title="Ubuntu logo pack zip file">Ubuntu logo pack [ZIP]&nbsp;&rsaquo;</a></li>
+            <li class="p-list__item is-ticked" style="background-image:url('data:image/svg+xml,%3Csvg xmlns=%22http://www.w3.org/2000/svg%22 width=%2214%22 height=%2214%22 viewBox=%220 0 14 14%22%3E%3Ccircle fill=%22%23333%22 cx=%227%22 cy=%227%22 r=%227%22/%3E%3Cpath fill=%22%23fff%22 d=%22M6.1 10.813L2.41 8.105l1.184-1.613L5.9 8.187l4.393-4.394 1.414 1.414z%22 /%3E%3C/svg%3E');"><a href="https://assets.ubuntu.com/v1/47b9260c-canonical-brandmark1.zip" title="Canonical logo pack">Canonical logo pack [ZIP]&nbsp;&rsaquo;</a></li>
```

Comment:
Should we update these two zip files with the new brand assets? A question for @waynecrosby I think.

Reply:
He needs to take a look before we release new assets